cli/src/commands/random-data/index.ts

```diff
@@ -34,6 +34,11 @@ export default class RandomData extends BaseCommand {
     }
 
     const { tables } = branchDetails.schema;
+    if (tables.length === 0) {
+      this.warn(
+        'Your database has no tables. To create one, use `xata schema edit`. Once your database has at least one table, running this command again will generate random data for you.'
```

Comment:
```suggestion
        'Your database has no tables. To create one, use `xata schema edit`. Once your database has at least one table, running this command again will generate random data for you.\n'
```

Let's make space for the next output.